templates/service_templates/redis.service.erb

```diff
@@ -13,9 +13,9 @@ Type=forking
 ExecStart=/usr/bin/redis-server <%= @redis_file_name %>
```

Comment:
I wondered if this shouldn't also use the variable, but then I realized we should drop 16.04: https://github.com/voxpupuli/puppet-redis/pull/406